src/main/java/edu/cornell/kfs/module/purap/batch/service/impl/JaggaerUploadFileServiceImpl.java

```diff
@@ -199,13 +199,9 @@ private URI buildSupplierUploadURI() {
         }
     }
 
-    private void generateResultsReport(List<JaggaerUploadFileResultsDTO> resultsList) {
-        /*
-         * A full report will be implemented on Jira KFSPTS-29084. Until then, this
-         * will produce a nice summary at the bottom of the log file
-         */
+    private void logUploadFileResusts(List<JaggaerUploadFileResultsDTO> resultsList) {
```

Comment:
Minor typo in method name: `...FileResusts` --> `...FileResults`

Also, is this method supposed to be converted into a report-generating one? Or has that work been deferred to a future ticket?

Reply:
I switched the name to logUploadFileResults

The functionals currently believe they don't need this particular report.  Jaggaer generates a report internally if there are errors on the upload.  We should continue to log the results to the job run log to aid in trouble shooting if there are upload errors.

Reply:
Ok, thanks for the explanation.